package.json

```diff
@@ -103,7 +104,7 @@
     "release-branch": "./scripts/release-branch.sh",
     "test": "yarn lint && yarn test-only",
     "test-ci": "yarn build && yarn test-only",
-    "test-only": "node --max_old_space_size=4096 ./node_modules/.bin/jest --coverage --verbose",
+    "test-only": "cross-env NODE_OPTIONS=--max_old_space_size=4096 node_modules/.bin/jest  --coverage --verbose",
```

Comment:
would `"test-only": "node --max_old_space_size=4096 node_modules/jest/bin/jest.js  --coverage --verbose",` be easier?